asciih.c

```diff
@@ -1,8 +1,9 @@
 #include <stdio.h>
 
-int main(int argc,char **argv)
+int main(int argc, char** argv)
 {
-  FILE *f=fopen("ascii.h","w");
-  for(int i=0;i<256;i++) fprintf(f,"#pragma charmap (0x%02x,0x%02x)\n",i,i);
+  FILE* f = fopen("ascii.h", "wt");
+  for (int i = 0; i < 256; i++)
```

Comment:
I'm not sure Linux/Unix boxes  do anything meaningful with the "t" mode

Reply:
Right, linux will ignore 'b' and 't'. The gotchya is that if left unspecified, then on windows machines, it will default to 't'. That causes windows users some grief if a binary file was intended, as it decides to:
- do EOL conversions (whenever it sees a 0x0A (\n), it replaces it with 0x0D 0x0A (\r\n).
- stops reading a file when it sees an EOF char (or is it EOT? I forget exactly)

So this issue caused us grief in relation to this card, as adtbm noticed that the thumbnail files generated by our Jenkins builds were corrupted. Upon investigation, this turned out to be due to the Jenkins build agents being Windows machines.